packages/design/src/_util/useFlexGapSupport.ts

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { detectFlexGapSupported } from './styleChecker';
 
 export default () => {
-  const [flexible, setFlexible] = React.useState(false);
+  const [flexible, setFlexible] = React.useState(true);
```

Comment:
if flexible is `false` by default, flex gap and compatible gap style maybe work for a moment. It will cause `double gap` shrink to `single gap`.